array-methods/4-array.js

```diff
@@ -28,6 +28,13 @@ var data = [
 
 // write a function that will sum all of the dogs ages in dog years using for loop.
 // 1 human year = 7 dog year
+function dog_year(dogs){
+  var count = 0;
+  for(var i=0; i<dogs.length; i++){
+     count += dogs[i].age;
+  }
+  var dogYear = count*7;
+}
```

Comment:
Naming conventions follow this: https://github.com/airbnb/javascript#naming-conventions